.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Issue
+about: Bug or feature request
+labels: v3
```

Comment:
Seems almost all the bugs are v3, so I thought this might help. We can remove it, though.